changelog/20663.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+core (enterprise): support reloading configuration for automated reporting via SIGHUP
+```
```

Comment:
Do you think it would be worth noting the change in behavior for the activitylog as well? Maybe something like "license changes on reload result in restarting the activitylog"?

You can add a second release note in the same changelog entry if you think it's worthwhile.